guides/rails_guides/generator.rb

```diff
@@ -122,16 +128,34 @@ def select_only(guides)
 
       def process_scss
         system "bundle exec dartsass \
-          #{@guides_dir}/assets/stylesrc/style.scss:#{@output_dir}/stylesheets/style-v2.css \
-          #{@guides_dir}/assets/stylesrc/highlight.scss:#{@output_dir}/stylesheets/highlight-v2.css \
-          #{@guides_dir}/assets/stylesrc/print.scss:#{@output_dir}/stylesheets/print-v2.css"
+          #{@guides_dir}/assets/stylesrc/style.scss:#{@output_dir}/stylesheets/style.css \
+          #{@guides_dir}/assets/stylesrc/highlight.scss:#{@output_dir}/stylesheets/highlight.css \
+          #{@guides_dir}/assets/stylesrc/print.scss:#{@output_dir}/stylesheets/print.css"
       end
 
       def copy_assets
         source_files = Dir.glob("#{@guides_dir}/assets/*").reject { |name| name.include?("stylesrc") }
         FileUtils.cp_r(source_files, @output_dir)
       end
 
+      def add_digests
+        @assets_with_digest = {}
```

Comment:
We should initialize this hash even if `dry_run?` is true. That is why the tests are failing.

Reply:
Ah thanks! I missed that, it's fixed now.